crypto/evp/p_lib.c

```diff
@@ -1123,8 +1123,11 @@ static int print_set_indent(BIO **out, int *pop_f_prefix, long *saved_indent,
 
         *saved_indent =  (i < 0 ? 0 : i);
         if (BIO_set_indent(*out, indent) <= 0) {
-            if ((*out = BIO_push(BIO_new(BIO_f_prefix()), *out)) == NULL)
+            BIO *prefbio = BIO_new(BIO_f_prefix());
+
+            if (prefbio == NULL)
                 return 0;
+            *out = BIO_push(prefbio, *out);
```

Comment:
Not sure this is good form to rely on internal details of the function.. I know BIO_push() only fails if it is passed a NULL, bu tit is probably safer to check for NULL.

Reply:
BIO_push is documented like this: `BIO_push() returns the head of the chain, which usually is b, or next if b is NULL.` There is no place for NULL error return in the documentation.